### PR TITLE
fix label autcomplete with checkmk 2.2.0

### PR DESCRIPTION
**Pull request by BenediktSeidl**
from CMK-12680

### CODE REVIEW

File: tests/unit/ui/QueryEditor.test.tsx [hunk 1]
```diff
@@ -138,72 +134,89 @@ describe('QueryEditor RAW', () => {
...
+    expect(contextAutocomplete).toHaveBeenCalledWith(
+      'available_graphs',
+      expect.anything(),
+      expect.anything(),
+      expect.anything()
     );
-    autocompleterRequest.mockClear();
+    contextAutocomplete.mockClear();
 
     const hostInput = screen.getByLabelText('Hostname');
```
Comment (oKenneth) [inline tests/unit/ui/QueryEditor.test.tsx:186]: Should screen be replaced with r here as well? As you did in line 161/162

File: tests/unit/ui/QueryEditor.test.tsx [hunk 2]
```diff
@@ -138,72 +134,89 @@ describe('QueryEditor RAW', () => {
...
+    expect(contextAutocomplete).toHaveBeenCalledWith(
+      'available_graphs',
+      expect.anything(),
+      expect.anything(),
+      expect.anything()
     );
-    autocompleterRequest.mockClear();
+    contextAutocomplete.mockClear();
 
     const serviceInput = screen.getByLabelText('Service');
```
Comment (oKenneth) [inline tests/unit/ui/QueryEditor.test.tsx:204]: same as above

**Review by oKenneth — COMMENTED** [review]
I'm unsure if the `screen` var should be replaced by the newly introduced var `r` in all places or if not so...?

File: tests/unit/ui/QueryEditor.test.tsx (hunk 1 above)
Comment (BenediktSeidl) [reply tests/unit/ui/QueryEditor.test.tsx:186]: i changed it back to screen, in favor of a smaller patch.

**Review by oKenneth — APPROVED** [review]

**Pull request merged**